### PR TITLE
Update documentation & revision guidance

### PR DESCRIPTION
**Pull request by dhimmel**
Refs https://github.com/greenelab/deep-review/issues/678

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,101 +7,59 @@
...
-#### Current Status - Submitted
+### Current stage: revisions
 
-We have submitted the manuscript to the journal. A preprint is available at
-[bioRxiv](https://doi.org/10.1101/142760). We are still accepting contributions
-to improve this work. Feel free to create an issue, contribute some text via a
-pull request, and pitch in. Authorship criteria remain the same.
+We are currently working on manuscript revisions in response to two external peer reviews.
+See [Issue #678](https://github.com/greenelab/deep-review/issues/678) to coordinate this process.
+When addressing reviewer comments, please also update [`response-to-reviewers.md`](content/response-to-reviewers.md) as appropriate.
```
Comment (cgreene) [inline README.md]: Do you want to make this stronger? We'd like to not accept PRs on new topics unless they directly address review comments? Basically just "may" -> "will"

File: content/response-to-reviewers.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+Replace the TODO with the relevant response.
+Feel free to also break up paragraphs as necessary, but remember to keep the reviews formatted as blockquotes.
+
+***
+
+## Referee 1
+
+> The authors summarized over 400 literature references purely by narratives. The authors provided synopsis for each important reference, but lacks synthesis of related work. It would be better to synthesize related work into a table and analyze their characteristics. 
+
+TODO
```
Comment (cgreene) [inline content/response-to-reviewers.md]: For the ones that just say `todo`, do you want to add a link to the connected issue?

**Comment by dhimmel** [comment]
# Code Climate

@agitter and @cgreene have you found that code climate is doing anything useful for this repo? Is this something we should keep enabled but change the line length setting or disable entirely?

**Comment by cgreene** [comment]
Do you want to use this as an opportunity to disable line length checks via codeclimate?
https://github.com/markdownlint/markdownlint/blob/master/docs/RULES.md#md013---line-length
https://docs.codeclimate.com/v1.0/docs/disabling-individual-checks

**Comment by cgreene** [comment]
@dhimmel : now that we have travis, i am fine with disabling codeclimate - @agitter, do you agree?

**Comment by cgreene** [comment]
LGTM now 👍

File: README.md [hunk 3]
```diff
@@ -7,101 +7,59 @@
...
+At the moment, efforts are concentrated on revisions: content unrelated to revisions may be deferred or not accepted.
 
-#### More about the manuscript.
+**Manubot updates:**
```
Comment (agitter) [inline README.md]: Should we also remind contributors to pull the latest version of master before making revisions?  This was always the recommendation but will be even more important after #681.

File: README.md [hunk 4]
```diff
@@ -7,101 +7,59 @@
...
+### Current stage: revisions
 
-We have submitted the manuscript to the journal. A preprint is available at
-[bioRxiv](https://doi.org/10.1101/142760). We are still accepting contributions
-to improve this work. Feel free to create an issue, contribute some text via a
-pull request, and pitch in. Authorship criteria remain the same.
+We are currently working on manuscript revisions in response to two external peer reviews.
+See [Issue #678](https://github.com/greenelab/deep-review/issues/678) to coordinate this process.
+When addressing reviewer comments, please also update [`response-to-reviewers.md`](content/response-to-reviewers.md) as appropriate.
+At the moment, efforts are concentrated on revisions: content unrelated to revisions may be deferred or not accepted.
```
Comment (agitter) [inline README.md]: I also would like to focus on merging existing pull requests, but this text is okay.

File: content/01.abstract.md [hunk 5]
```diff
@@ -1,20 +1,11 @@
 ## Abstract {.page_break_before}
 
-Deep learning, which describes a class of machine learning algorithms, has
```
Comment (agitter) [inline content/01.abstract.md]: Should we try to merge some of the outstanding pull requests first?  That may take too long, so I'm okay updating the line wrapping now if you think that is most efficient.  We will need to provide help with merge conflicts for those pull requests though.

File: content/response-to-reviewers.md [hunk 6]
```diff
@@ -0,0 +1,48 @@
+# Response to Reviewers
```
Comment (agitter) [inline content/response-to-reviewers.md]: This is great 👍

File: content/01.abstract.md (hunk 5 above)
Comment (dhimmel) [reply content/01.abstract.md]: I only reflowed the abstract assuming that none of the PRs touch the abstract. Yes, we should focus on reviewing exiting PRs. They likely all have conflicts now. Tag me when it's time to merge and I will handle the conflict resolution. Sooner the better.

Comment (agitter) [reply content/01.abstract.md]: @dhimmel ah, I see.  I reviewed this hastily between meetings.

**Pull request merged**